command/agent/host/linux.go

```diff
@@ -0,0 +1,42 @@
+// +build linux
+
+package host
+
+import (
+	"bufio"
+	"os"
+	"strings"
+)
+
+// mountedPaths produces a list of mounts
+func mountedPaths() []string {
+	fh, err := os.Open("/proc/mounts")
+	if err != nil {
+		return []string{err.Error()}
+	}
+	rd := bufio.NewReader(fh)
+
+	var paths []string
+	for {
+		str, err := rd.ReadString('\n')
+		if err != nil {
+			break
+		}
+
+		ln := strings.Split(str, " ")
+		switch ln[2] {
+		case "autofs", "binfmt_misc", "cgroup", "debugfs",
+			"devpts", "devtmpfs",
+			"fusectl", "fuse.lxcfs",
+			"hugetlbfs", "mqueue",
+			"proc", "pstore", "rpc_pipefs", "securityfs",
+			"sysfs", "tmpfs", "vboxsf":
+			continue
+		default:
+		}
+
+		paths = append(paths, ln[1])
+	}
+
+	return paths
+}
```

Comment:
I think this is our 3rd implementation of mount point enumeration:

1. Used in `/v1/client/stats` - https://github.com/hashicorp/nomad/blob/v0.12.0-beta2/client/stats/host.go#L183-L209
2. Used in client node fingerprinting - https://github.com/hashicorp/nomad/blob/v0.12.0-beta2/client/fingerprint/storage_unix.go#L16

We don't have to fix this up for 0.12.0, but would you mind investigating if there's an opportunity to share code here and file an issue if so?